src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -47,6 +54,10 @@
       <Name>ServicesVisualStudio.Next</Name>
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Ngen>true</Ngen>
```

Comment:
@jasonmalinowski that was my thought as well.  This PR htough is bascially holding up all insertions.  I'd rather get it in soon and do clean up work after if it's expensive.